tests/features/site.feature

```diff
@@ -11,7 +12,8 @@ Feature: site
 
   Scenario: Site Connection Mode
     @vcr site-connection-mode
-    Given a site named "[[test_site_name]]"
+    Given I am authenticated
+    And a site named "[[test_site_name]]"
     When I run "terminus site connection-mode --site=behat-test --env=dev"
```

Comment:
While that makes sense, isn't this file appearing in this PR *because* its behavior had been changed? Should this file be excluded from the PR?